mods/cnc/maps/gdi09/rules.yaml

```diff
@@ -0,0 +1,150 @@
+World:
+	LuaScript:
+		Scripts: campaign-global.lua, gdi09.lua, gdi09-AI.lua
+	MusicPlaylist:
+		StartingMusic: march
+		VictoryMusic: gdi_win1
+	MissionData:
+		Briefing: Take out Nod turrets along shore so Gunboats can move in safely on the Nod base.\n\nThe Nod base must be destroyed.\n\nIf gunboats can get in, they should be able to destroy the base with no difficulty.\n\nKeep an eye out for the new weapon Nod is rumored to be working on.
+		BriefingVideo: gdi9.vqa
+		WinVideo: gunboat.vqa
+		LossVideo: gameover.vqa
+
+ATWR:
+	Buildable:
+		Prerequisites: ~disabled
+
+NUK2:
+	Buildable:
+		Prerequisites: ~disabled
+
+HPAD:
+	Buildable:
+		Prerequisites: ~disabled
+
+BRIK:
+	Buildable:
+		Prerequisites: ~disabled
+
+EYE:
+	Buildable:
+		Prerequisites: ~disabled
+
+GUN:
+	Buildable:
+		Prerequisites: ~disabled
+
+OBLI:
+	Buildable:
+		Prerequisites: ~disabled
+
+TMPL:
+	Buildable:
+		Prerequisites: ~disabled
+
+HTNK:
+	Buildable:
+		Prerequisites: ~disabled
+
+TRAN:
+	Buildable:
+		Prerequisites: ~disabled
+
+ORCA:
+	Buildable:
+		Prerequisites: ~disabled
+
+RMBO:
+	Buildable:
+		Prerequisites: ~disabled
+
+MSAM:
+	Buildable:
+		Prerequisites: ~disabled
+
+MCV:
+	Buildable:
+		Prerequisites: ~disabled
+
+BOAT:
+	Buildable:
+		Prerequisites: ~disabled
+
+FTNK:
+	Buildable:
+		Prerequisites: ~disabled
+
+STNK:
+	Buildable:
+		Prerequisites: ~disabled
+
+HELI:
+	Buildable:
+		Prerequisites: ~disabled
+
+LTNK:
+	Buildable:
+		Prerequisites: ~afld
+
+ARTY:
+	Buildable:
+		Prerequisites: ~afld
+
+E4:
+	Buildable:
+		Prerequisites: barracks
+
+E5:
+	Buildable:
+		Prerequisites: ~disabled
+
+MLRS:
+	Buildable:
+		Prerequisites: ~disabled
+
+SAM:
+	Buildable:
+		Prerequisites: ~disabled
+	SpawnActorOnDeath:
+		Actor: e1
```

Comment:
What is the idea here?

Reply:
In the original there was a chance for a sam site to spawn an e1 on destruction, if I remeber correctly. Should we scratch this here?

Reply:
Imho yes, we don't spawn actors out of any of the other buildings as well (original RA did this for all buildings, so I guess original TD could have been the same).